example/spring-batch/src/main/kotlin/com/linecorp/kotlinjdsl/example/spring/batch/entity/employee/EmployeeDepartment.kt

```diff
@@ -0,0 +1,35 @@
+package com.linecorp.kotlinjdsl.example.spring.batch.entity.employee
+
+import com.linecorp.kotlinjdsl.example.spring.batch.annotation.CompositeId
+import jakarta.persistence.*
+import java.io.Serializable
+import java.util.*
+
+@Entity
+@Table(name = "employee_department")
+@IdClass(EmployeeDepartment.EmployeeDepartmentId::class)
+class EmployeeDepartment(
+    @Id
+    @ManyToOne
+    @JoinColumn(name = "employee_id")
+    val employee: Employee,
+
+    @Id
+    @Column(name = "department_id")
+    val departmentId: Long,
+) {
```

Comment:
I've implemented this Entity incorrectly. Please update the develop/3.0.0 branch and use the latest implementation.

Reply:
I've updated Entity implementations.